src/api/app/controllers/source_controller.rb

```diff
@@ -685,15 +685,15 @@ def project_command_copy
       end
     end
 
+    job_params = params.slice(
+      :cmd, :user, :comment, :oproject, :withbinaries, :withhistory, :makeolder, :makeoriginolder, :noservice
+    ).permit!.to_h
+    job_params[:user] = User.session.id
```

Comment:
would be possible to have the `nobody id` here?

Reply:
`User.session` never returns nobody - it would crash on nobody. But nobody is excluded from the controller. But I can make `User.session!` to make it more obvious